app/assets/javascripts/refresh_token.js.coffee

```diff
@@ -14,9 +14,9 @@
 
 $ ->
   refreshToken = ->
-    token = currentPlayer.media.src.split('?')[1]
```

Comment:
This doesn't seem to work any more (`currentPlayer.media.src` returns a bogus URL)